projects/plugins/super-cache/tests/e2e/package.json

```diff
@@ -23,17 +23,17 @@
 		"test:run": "jest --runInBand"
 	},
 	"devDependencies": {
-		"@jest/globals": "29.1.1",
+		"@jest/globals": "^28.1.0",
 		"@typescript-eslint/eslint-plugin": "5.36.1",
 		"@typescript-eslint/parser": "5.36.1",
 		"axios": "0.27.2",
 		"cheerio": "1.0.0-rc.12",
 		"dotenv": "16.0.2",
 		"eslint": "8.22.0",
 		"eslint-plugin-jest": "26.8.7",
-		"jest": "^29.0.3",
+		"jest": "^28.1.0",
 		"shell-escape": "0.2.0",
-		"ts-jest": "^29.0.2",
+		"ts-jest": "28.0.8",
 		"typescript": "4.8.2",
 		"zod": "3.19.1"
```

Comment:
This also seems unused.

Reply:
That's ts-jest, not zod.

Reply:
Oh, right - sorry. Funny how the green line draws the eye more than the line immediately above your comment. 😊

Yeah, zod was used during an experimental bit of code which I removed, so yes it needs to go. Deleted in 22452b2865

Thanks for the re-review, and apologies for not noticing it for a few days.